modules/nixos/common.nix

```diff
@@ -24,5 +24,7 @@
   # Just disable it since we are using telegraf to monitor raid health.
   systemd.services.mdmonitor.enable = false;
 
+  environment.systemPackages = [ pkgs.git ];
```

Comment:
Added `gitMinimal` in https://github.com/numtide/srvos/commit/9f9073c808598b2584d509bfaab6a58e20d95ce1 so this can be dropped.